package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "nucypher-contracts",
+  "version": "0.0.1",
+  "license": "AGPL-3.0-or-later",
+  "description": "NuCypher Network Smart Contracts",
```

Comment:
Side note: we may want to change this to `NuCypher Smart Contracts` or something like that without the `NuCypher Network` combination since we are now Threshold.